components/phpunit_bridge.rst

```diff
@@ -207,10 +207,10 @@ Troubleshooting
 ~~~~~~~~~~~~~~~
 
 The ``@group time-sensitive`` works "by convention" and assumes that the
-namespace of the tested class can be obtained just by removing the ``\Tests\``
+namespace of the tested class can be obtained just by removing the ``Tests\``
```

Comment:
@teohhanhui This is the only change that must be revert, right? The fixes below are still valid if I don't miss anything.